apis/extension/slo_controller_config.go

```diff
@@ -34,6 +34,13 @@ const (
 	SystemConfigKey            = "system-config"
 )
 
+// +k8s:deepcopy-gen=true
+type NodeCfgProfile struct {
+	Name string `json:"name,omitempty"`
```

Comment:
Perhaps we need to print the name of the node config for the config merged failed. Otherwise, the field makes sense only when shown to end users.

Reply:
we'd better add some comments about preserving this field